library/dnet.c

```diff
@@ -314,6 +314,31 @@ int dnet_send_reply(void *state, struct dnet_cmd *cmd, const void *odata, unsign
 	return err;
 }
 
+void dnet_queue_wait_threshold(void *state)
+{
+	struct dnet_net_state *st = state;
```

Comment:
`struct dnet_net_state` is private internal structure, it can not be operated on by backends, which pass it as a void pointer.
It is the same logic as with `dnet_send_reply()` and friends